### ISSUE TITLE
Improper negative controls

### ISSUE THREAD

**Issue by chandryou** [cukarthik/IUDEstimationStudy#1]
Levonorgestrel and Copper can have some systemic effect. We need to consider this more. 

I think some of negative controls are inappropriate as below. 

Concept Id | Concept Code | Name | Vocabulary|Reason
-- | -- | -- | -- | ----------
26662 | 111551000 | Testicular hypofunction | SNOMED|It doesn't happen in women |
432883 | 191611001 | Recurrent major depressive episodes,   moderate | SNOMED | [Ref](https://www.health.harvard.edu/blog/can-hormonal-birth-control-trigger-depression-2016101710514)
438409 | 406506008 | Attention deficit hyperactivity disorder | SNOMED|This disease is occurred in childeren

**Comment by vojtechhuser** [comment]
chandryou is probably right

**Comment by mattspotnitz** [comment]
I agree that testicular hypofunction is not appropriate and has been removed.  However, I am not as convinced that depression or ADHD need to be removed from the negative control list. They are diagnoses that women could have antecedent to IUD placement and should have no effect on choice of device.

**Comment by chandryou** [comment]
@mattspotnitz It's on you, of course. I'm not sure ADHD or depression must be removed. But if I were you, I would remove them because they might be associated with these devices.  

But, just to be clear 

Negative controls are falsification endpoints. It doesn't matter whether they're associated with treatment assignment or not. 

https://ohdsi.github.io/TheBookOfOhdsi/MethodValidity.html
